### DISCUSSION TITLE
【要望】Q&Aの回答に「順番」を付加する

### DISCUSSION THREAD

**Discussion by nak435** [increments/qiita-discussions#556] [要望・アイデア・不具合]
Q&Aに複数の回答がついているとき、「いいね」やコメントが追加されると、回答の表示順序そのものが変化すると思います。
そこで、そもそも、どの順序で回答されたのか、オリジナルの回答順序を示す数字をつけると、読み手に便利だと思いました。
ご検討いただけたら幸いです。よろしくお願いします。

**Comment by tanaka-asami415** [comment]
@nak435 
ご意見ありがとうございます！

ニーズとしては、「回答された順番で確認したい」ということだと理解いたしました。
「回答を時系列順に表示したい」というご意見もいただいており（ #492 ）
時期はお約束できませんが、並び替えの方法で検討したいと思います🙏

**Reply by nak435** [reply]
ご確認ありがとうございます。
「順番を確認したい」だけで「並び替える」着想はありませんでしたが、確かに「回答順に表示」できれば、自分の要望は満たされます。
よろしくお願いします。
